app/Http/Controllers/Api/ApiReminderController.php

```diff
@@ -153,4 +155,28 @@ public function reminders(Request $request, $contactId)
 
         return ReminderResource::collection($reminders);
     }
+
+    /**
+     * Get the reminders for the month given in parameter.
+     * - 0 means current month
+     * - 1 means month+1
+     * - 2 means month+2...
+     *
+     * @param  Request  $request
+     * @param  int  $month
+     * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection|\Illuminate\Http\JsonResponse
+     */
+    public function upcoming(Request $request, int $month = 0)
```

Comment:
We need to check `$month` is between 0 and 3.

Reply:
As I imagine, no limit for `$month` variable. Limit only for `dashboard` widget. But this is API, so feel free to enter what value you want

Reply:
I wasn't sure at start, but let's do it, ok!